src/Prism/Mvvm/ErrorsContainer.cs

```diff
@@ -26,12 +24,7 @@ public class ErrorsContainer<T>
         /// event.</param>
         public ErrorsContainer(Action<string> raiseErrorsChanged)
         {
-            if (raiseErrorsChanged == null)
-            {
-                throw new ArgumentNullException(nameof(raiseErrorsChanged));
-            }
-
-            this.raiseErrorsChanged = raiseErrorsChanged;
+            this.raiseErrorsChanged = raiseErrorsChanged ?? throw new ArgumentNullException(nameof(raiseErrorsChanged));
```

Comment:
private fields should start with a _, so this won't be necessary if the fields follow the guidance.